src/routes/amt/getVersion.test.ts

```diff
@@ -106,23 +16,26 @@ describe('version', () => {
     resSpy.send.mockReturnThis()
 
     devices['4c4c4544-004b-4210-8033-b6c04f504633'] = new ConnectedDevice(null, 'admin', 'P@ssw0rd')
-    versionSpy = jest.spyOn(devices['4c4c4544-004b-4210-8033-b6c04f504633'], 'getVersion')
+    softwareIdentitySpy = jest.spyOn(devices['4c4c4544-004b-4210-8033-b6c04f504633'], 'getSoftwareIdentity')
+    setupAndConfigurationServiceSpy = jest.spyOn(devices['4c4c4544-004b-4210-8033-b6c04f504633'], 'getSetupAndConfigurationService')
   })
 
   it('should get version', async () => {
-    versionSpy.mockResolvedValueOnce(versionResponse)
+    softwareIdentitySpy.mockResolvedValueOnce(softwareIdentityResponse)
+    setupAndConfigurationServiceSpy.mockResolvedValueOnce(setupAndConfigurationServiceResponse)
     await version(req, resSpy)
     expect(resSpy.status).toHaveBeenCalledWith(200)
-    expect(resSpy.json).toHaveBeenCalledWith(versionResponse)
+    expect(resSpy.json).toHaveBeenCalledWith(Versionresponse)
```

Comment:
Was this meant to be changed? should be `versionResponse` to follow naming convention

Reply:
done